pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/Utils.groovy

```diff
@@ -292,6 +293,23 @@ class Utils {
         return cause != null
     }
 
+    /**
+     * Check if this run was caused by a cause.
+     */
+    @Whitelisted
```

Comment:
Turns out we don't need these `@Whitelisted` annotations - they're from the long ago before plugin-provided Pipeline code became exempt from whitelist checks, so might as well remove this.